aclk/aclk.h

```diff
@@ -14,7 +14,7 @@
 #endif /* ENABLE_ACLK */
 
 extern int aclk_connected;
-extern int use_mqtt_5;
+extern int aclk_alert_reloaded;
```

Comment:
Hi Timo! I think we don't need this... ?

Reply:
Thanks, rebase mistake